nextflow.config

```diff
@@ -271,11 +271,11 @@ dag {
 
 manifest {
     name            = 'nf-core/phaseimpute'
-    author          = """LouisLeNezet"""
+    author          = """Louis Le Nezet, Anabella Trigila"""
     homePage        = 'https://github.com/nf-core/phaseimpute'
     description     = """Phasing and imputation pipeline"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=24.04.4'
+    nextflowVersion = '!>=24.04.2'
```

Comment:
why 24.04.2 ?

Reply:
We updated it before, but the template say 24.04.2